### ISSUE TITLE
Unable to boot

### ISSUE THREAD

**Issue by fender357** [rg35xx-cfw/Koriki#15]
### What happened?

I've been attempting to boot Koriki for the first time with no success. I've tried two previous builds before 0.8 (with and without the bootfix files), and across all three, the result has been the same. 
When I press the power button for 3-4 seconds, I get "den)( software engineering" logo, and then nothing after that.  Using the stock OS SD card that came with the RG35XX works fine. 
I am using a 16GB SanDisk Ultra for the OS sd card. 
I also have a 512GB SanDisk Ultra for the secondary card. I have tried with and without this secondary card inserted and the result is the same.
What I believe may be the serial number (located on the packaging) shows "X003LQZ2WR" in case that matters for possible hardware variations.

**Comment by fender357** [comment]
Maybe this issue can be closed out, but for anyone else experiencing this. It seems my device was low on power. Once I plugged into power I was able to get Koriki to boot up. Other things to note from my testing and reading through other instructions. I could not get the unit to boot at all with no SD cards inserted. I have to insert the stock OS to get the system information to show. The firmware version shown under the Ambernic system details is V1.20221223.
